app/models/user_identifier.rb

```diff
@@ -11,6 +11,7 @@
 #
 # Indexes
 #
+#  fk_rails_fe95df7db0                (identifier_scheme_id)
```

Comment:
Is the UserIdentifier model sticking around for a release to migrate the objects over to the new Identifier model?

Reply:
yep. need it for the migration of data unfortunately